packages/search/addon/services/navi-search/navi-asset-search-provider.js

```diff
@@ -0,0 +1,131 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * This service is used to search for reports and dashboards stored in the persistence layer.
+ */
+
+import { inject as service } from '@ember/service';
+import NaviBaseSearchProviderService from '../navi-base-search-provider';
+import { keepLatestTask } from 'ember-concurrency-decorators';
+import { pluralize } from 'ember-inflector';
+
+export default class NaviAssetSearchProviderService extends NaviBaseSearchProviderService {
+  /**
+   * @property {Ember.Service} store
+   */
+  @service store;
+
+  /**
+   * @property {Ember.Service} user
+   */
+  @service user;
+
+  /**
+   * @property {String} _displayComponentName
+   * @private
+   */
+  _displayComponentName = 'navi-search-result/asset';
```

Comment:
they could be placed anywhere, I'm putting them under this directory just for organization purposes and because it looks good instantiating them by `NaviSearchResult::ComponentName` 😝

Reply:
ok. I thought the plan was to place all the services and components in specific directories.